### ISSUE TITLE
Bug with Trans component without children

### ISSUE THREAD

**Issue by SPAHI4** [i18next/i18next-parser#84]
1.0.0-beta6
cli

I'm trying to use `Trans` component as self-closing tag.

`<Trans i18nKey="Foo" />`

 CLI just do nothing without any error.

`<Trans i18nKey="Foo"></Trans>`

This works fine

**Comment by karellm** [comment]
Thanks for the bug report. I've just published `1.0.0-beta7` that address that issue.

**Comment by SPAHI4** [comment]
@karellm I still have this error on `1.0.0-beta7`

**Comment by karellm** [comment]
@SPAHI4 Can you detail this please. Here is the [test](https://github.com/i18next/i18next-parser/blob/master/test/parser.test.js#L173) and the associated [template file](https://github.com/i18next/i18next-parser/blob/master/test/templating/react.jsx#L14). A reproducible failing test case or more details would help.

**Comment by SPAHI4** [comment]
@karellm 

the code below, `templating/react.jsx`
```javascript
import React from 'react'
import { translate, Trans, Interpolate } from 'react-i18next'

const bar = () => (
  <div>
    <span><Trans i18nKey="bar"></Trans></span>
  </div>
);

const foo = () => (
  <div>
    <span><Trans i18nKey="foo" /></span>
  </div>
);

class Test extends React.Component {
  render () {
    const { t } = this.props
    return (
      <div>
        <h1>{t('first')}</h1>
        <Interpolate i18nKey="second" value="some thing" component={interpolateComponent} />
        <Trans i18nKey="third.first" count={count}>
            Hello <strong title={t('fourth')}>{{name}}</strong>, you have {{count}} unread message. <Link to="/msgs">Go to messages</Link>.
        </Trans>
        <Trans i18nKey="fifth" count={count} />
      </div>
    )
  }
}

export default translate('react')(Test)
```

test, look at `foo` value
![image](https://user-images.githubusercontent.com/1670998/38058646-46ba70f8-32ec-11e8-83a7-a9257927c180.png)

**Comment by karellm** [comment]
Thanks, it helped. I just published `1.0.0-beta8` that should fix it for good.
